Python/Product/Debugger/Debugger/DebugEngine/Remote/PythonRemoteDebugPortSupplier.cs

```diff
@@ -97,6 +103,7 @@ public class PythonRemoteDebugPortSupplier : IDebugPortSupplier2, IDebugPortSupp
         }
 
         public int GetPortSupplierName(out string pbstrName) {
+            // TODO: Localization - translate or not?
             pbstrName = "Python remote (ptvsd)";
```

Comment:
I think so, but then again, 2/3 of the string won't be localized anyway. Leave it for now